### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by blackrez**
- remove dep section
- add modules suport
- remove old stuff about organisation migration
- upgrade golang and gorgonia version
- fix contribiting link
- add slack channel
- add sections about hardware and os supported

@chewxy What is the statement about supported golang version in travis, there is only 1.12 and 1.13 tested? 
@owulveryck  Can I add GoMachines in documentation ?

### CODE REVIEW

**Comment by chewxy** [comment]
The supported statement is technically 2 back (meaning 1.11, 1.12, and 1.13). But Gonum only supports 1.12+ for now, so we'll have to stick with that

File: README.md [hunk 1]
```diff
@@ -43,9 +54,21 @@ We use [semver 2.0.0](http://semver.org/) for our versioning. Before 1.0, Gorgon
...
+Gorgonia supports 2 versions below the Master branch of Go. This means Gorgonia will support the current released version of Go, and up to 4 previous versions - providing something doesn't break. Where possible a shim will be provided (for things like new `sort` APIs or `math/bits` which came out in Go 1.9).
+
+The current version of Go is 1.13.1. The earliest version Gorgonia supports is Go 1.11.x but Gonum supports only 1.12+.
+
+## Hardware and OS supported ##
 
-The current version of Go is 1.9.2. The earliest version Gorgonia supports is Go 1.6.x. 
+Gorgonia runs on :
+- linux/AMD64
+- linux/ARM7
```
Comment (blackrez) [inline README.md]: I should add there is no hardware accelaration on ARM.

File: README.md [hunk 2]
```diff
@@ -43,9 +54,21 @@ We use [semver 2.0.0](http://semver.org/) for our versioning. Before 1.0, Gorgon
 
 ## Go Version Support ##
 
-Gorgonia aims to support 5 versions below the Master branch of Go. This means Gorgonia will support the current released version of Go, and up to 4 previous versions - providing something doesn't break. Where possible a shim will be provided (for things like new `sort` APIs or `math/bits` which came out in Go 1.9).
+Gorgonia supports 2 versions below the Master branch of Go. This means Gorgonia will support the current released version of Go, and up to 4 previous versions - providing something doesn't break. Where possible a shim will be provided (for things like new `sort` APIs or `math/bits` which came out in Go 1.9).
+
+The current version of Go is 1.13.1. The earliest version Gorgonia supports is Go 1.11.x but Gonum supports only 1.12+.
```
Comment (owulveryck) [inline README.md]:
```suggestion
The current version of Go is 1.13.1. The earliest version Gorgonia supports is Go 1.11.x but Gonum supports only 1.12+. Therefore, the minimum Go version to run the master branch is Go > 1.12.
```

File: README.md (hunk 1 above)
Comment (owulveryck) [reply README.md]: We should probably add a different section about hardware acceleration.

**Comment by owulveryck** [comment]
For GoMachine, IMHO, it's better to wait for a more mature version or Gorgonia v0.10.0.

**Pull request merged**